### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by Shrey0610** [recodehive/resume-pitch#100] [Q&A]
Name: Shrey Shah
LinkedIn: www.linkedin.com/in/shrey-shah-06o10
Resume:  [Shrey Shah Resume.pdf](https://github.com/user-attachments/files/17044896/Shrey.Shah.Resume.pdf)
GitHub Link: https://github.com/Shrey0610

**Comment by sanjay-kv** [comment]
[shrey s.pdf](https://github.com/user-attachments/files/17089497/shrey.s.pdf)

[ATS Version of CV (28).docx](https://github.com/user-attachments/files/17089498/ATS.Version.of.CV.28.docx)
@Shrey0610
